src/components/ChannelList/ChannelList.tsx

```diff
@@ -112,47 +341,39 @@ const ChannelList = (props) => {
     status,
   } = usePaginatedChannels(
     client,
-    props.filters || DEFAULT_FILTERS,
-    props.sort || DEFAULT_SORT,
-    props.options || DEFAULT_OPTIONS,
+    filters || DEFAULT_FILTERS,
+    sort || DEFAULT_SORT,
+    options || DEFAULT_OPTIONS,
     activeChannelHandler,
   );
 
-  const loadedChannels = props.channelRenderFilterFn
-    ? props.channelRenderFilterFn(channels)
+  const loadedChannels = channelRenderFilterFn
+    ? channelRenderFilterFn(channels)
     : channels;
 
   useMobileNavigation(channelListRef, navOpen, closeMobileNav);
 
   // All the event listeners
   useMessageNewListener(
     setChannels,
-    props.lockChannelOrder,
-    props.allowNewMessagesFromUnfilteredChannels,
-  );
-  useNotificationMessageNewListener(setChannels, props.onMessageNew);
-  useNotificationAddedToChannelListener(setChannels, props.onAddedToChannel);
-  useNotificationRemovedFromChannelListener(
-    setChannels,
-    props.onRemovedFromChannel,
-  );
-  useChannelDeletedListener(setChannels, props.onChannelDeleted);
-  useChannelHiddenListener(setChannels, props.onChannelHidden);
-  useChannelVisibleListener(setChannels, props.onChannelVisible);
-  useChannelTruncatedListener(
-    setChannels,
-    props.onChannelTruncated,
-    forceUpdate,
+    lockChannelOrder,
+    allowNewMessagesFromUnfilteredChannels,
   );
-  useChannelUpdatedListener(setChannels, props.onChannelUpdated, forceUpdate);
+  useNotificationMessageNewListener(setChannels, onMessageNew);
```

Comment:
don't all these guys need gennies passed in?

Reply:
nah, it gets inferred from the props passed into it